### PR TITLE
FAST: add support for project parents to bootstrap stage

### LINKED ISSUE #798: FAST core projects within folders
A lot of users are moving the FAST core projects (iac, billing, audit) are moved to their own folder(s) (e.g. a shared services folder). FAST should provide (or at least document) a method achieve this.

### PR DESCRIPTION
**Pull request by ludoo**
Fixes #798

### CODE REVIEW

File: fast/stages/00-bootstrap/variables.tf [hunk 1]
```diff
@@ -200,3 +200,18 @@ variable "prefix" {
     error_message = "Use a maximum of 9 characters for prefix."
   }
 }
+
+variable "project_parents" {
```
Comment (juliocc) [inline fast/stages/00-bootstrap/variables.tf:204]: parent_project_ids, wdyt?

**Review by juliocc — APPROVED** [review]
you also need to add the projectMover role to the bootstrap SA

File: fast/stages/00-bootstrap/variables.tf (hunk 1 above)
Comment (juliocc) [reply fast/stages/00-bootstrap/variables.tf:204]: I mean project_parent_ids

Comment (ludoo) [reply fast/stages/00-bootstrap/variables.tf:204]: yes on both counts :)

**Pull request merged**